### ISSUE TITLE
Dev: Versioning

### ISSUE THREAD

**Issue by JoeKae** [YarostheLaunchpadder/MissionPlanner-to-Litchi#17]
We should start to use the [Major.Minor.Patch](https://semver.org/lang/de/) version scheme.
[Bumpver](https://pypi.org/project/bumpver/) could be utilized.

**Comment by YarostheLaunchpadder** [comment]
This is a great idea, do you think it would be possible to implement Auto updates as well? For example, code that checks if there is a new release available on the releases page of this github repo, and then download the executable accordingly to the system it is ran on.

**Comment by JoeKae** [comment]
Yes that would be possible. Not sure if there is a package that does exactly this for us or if we have to implement this ourselves.
